clusterlib/tests/test_scheduler.py

```diff
@@ -4,19 +4,40 @@
 from __future__ import unicode_literals
 
 import os
+import os.path as op
+from time import sleep
 import subprocess
 from getpass import getuser
-from xml.etree.ElementTree import XMLParser, XML
 
 from nose.tools import assert_equal
 from nose.tools import assert_raises
 from nose.tools import assert_in
 from nose import SkipTest
 
-from ..scheduler import queued_or_running_jobs
-from ..scheduler import submit
-from ..scheduler import _which
-from ..scheduler import _get_backend
+from clusterlib.scheduler import queued_or_running_jobs
+from clusterlib.scheduler import submit
+from clusterlib.scheduler import _which
+from clusterlib.scheduler import _get_backend
+from clusterlib.testing import TemporaryDirectory
+
+
+def _do_dispatch(*args, **kwargs):
+    """Perform a dispatch with the submit command and return the job id"""
+    # TODO: This utility function should be properly documented any made more
+    # robust to be included in the scheduler module itself
```

Comment:
Do you often write script / launching script (as user) where you really want to know the job id? 
Thinking about this, I am not sure  that this should be made a public function, but this would be a great tool
for developing clusterlib.